.github/workflows/auto_release.yaml

```diff
@@ -0,0 +1,71 @@
+name: auto_release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    name: Publish GitHub release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Fetch all history for all tags and branches
+        run: git fetch --prune --unshallow
+      - name: Install gcc-mingw-w64-x86-64
+        run: |
+             sudo apt -q update
+             sudo apt -q install --no-install-recommends gcc-mingw-w64-x86-64
+      - name: Build truckersmp-cli.exe
+        run: make
+      - name: Strip truckersmp-cli.exe
+        run: x86_64-w64-mingw32-strip truckersmp-cli.exe
+      - name: Compress files
+        id: compress_files
+        run: |
+             tag="$(basename ${{ github.ref }})"
+             tardir="$(basename ${{ github.repository }})-${tag:1}"
+             mkdir -p .output/$tardir
+             mv * .output/$tardir
+             tar Jcvf output.tar.xz -C .output --owner=root --group=root $tardir
```

Comment:
Why are you specifically setting the owner and group to root, is that something common?
Unless `--same-owner` is given on extracting it would already have the extracting users ids set (`--no-same-owner` is the default). https://www.gnu.org/software/tar/manual/tar.html#SEC138

Reply:
I know that owner information won't be set when normal users unpack tarballs.
It seems that changing owner/group in tarballs is not "common"... Some projects set the information and others don't set the information:

```
$ xz -dc mesa-20.0.7.tar.xz | tar tfvv - | head -n 1
drwxr-xr-x dylan/dylan       0 2020-05-15 01:50 mesa-20.0.7/
$ xz -dc tar-1.32.tar.xz | tar tfvv - | head -n 1
drwxr-xr-x gray/gray         0 2019-02-23 21:57 tar-1.32/
$ xz -dc emacs-26.3.tar.xz | tar tfvv - | head -n 1
drwxr-xr-x nico/nico         0 2019-08-23 05:44 emacs-26.3/
$ xz -dc nano-4.9.2.tar.xz | tar tfvv - | head -n 1
drwxr-xr-x 1000/100          0 2020-04-07 18:29 nano-4.9.2/
$ xz -dc xz-5.2.5.tar.xz | tar tfvv - | head -n 1
drwxr-xr-x 0/0               0 2020-03-17 23:31 xz-5.2.5/
$ xz -dc gettext-0.20.2.tar.xz | tar tfvv - | head -n 1
drwxrwxr-x 0/0               0 2020-04-14 13:56 gettext-0.20.2/
```